### ISSUE TITLE
Cannot start Bayesian analytics LS agent with `Centos NodeJS` stack

### ISSUE THREAD

**Issue by sunix** [redhat-developer/rh-che#189]
The stack `CentOS NodeJS` doesn't have neither `node` nor `nodejs` command to start node js. At the moment, the agent detect these commands to start nodejs.

**Comment by l0rd** [comment]
NodeJS is installed but `scl enable` should be  prefixed:

```bash
[user@7fe940c2a264 projects]$ scl enable rh-nodejs4 'node --version'
v4.6.2
```

I beleive @dharmit is working to fix that in a way that prefixing with scl enable is not necessary anymore. @dharmit can you confirm? Is there an issue about that already?

**Comment by dharmit** [comment]
There's no dedicated issue about it except the one mentioned on https://github.com/redhat-developer/rh-che/issues/147. I'm working on removing this `scl enable` dependency.

**Comment by l0rd** [comment]
Fantastic thank you @dharmit

**Comment by ibuziuk** [comment]
Should be fixed by - https://github.com/eclipse/che-dockerfiles/pull/108, but still need to verify it

**Comment by mlabuda** [comment]
Commented in eclipse/che-dockerfiles#108. That solution won't work.

**Comment by JPinkney** [comment]
Fixed as of eclipse/che-dockerfiles@8e70296
